File: Java/spring-cloud-config/config-server-with-vault/src/main/java/jp/glory/config/server/SecurityConfig.java
package jp.glory.config.server;

import org.springframework.beans.factory.annotation.Value;
import org.springframework.context.annotation.Configuration;
import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
import org.springframework.security.config.annotation.web.builders.HttpSecurity;
import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;

@Configuration
public class SecurityConfig extends WebSecurityConfigurerAdapter {

    /**
     * デフォルト値は「password」のBCrpyptの結果。
     */
    @Value("${user.client.password:$2a$10$CQ84AulZgwTUFLoUQjHf/.mqrDw7inl7nNsYHG35jgbQuIzuftoZy}")
    private String password;

    @Override
    protected void configure(final AuthenticationManagerBuilder auth) throws Exception {
        auth.inMemoryAuthentication()
                .passwordEncoder(new BCryptPasswordEncoder())
                .withUser("client").password(password).roles("CLIENT");
    }

    @Override
    protected void configure(HttpSecurity http) throws Exception {
        http.httpBasic();
        http.authorizeRequests()
                .anyRequest().authenticated();

    }
}

File: Auth/OAuth2/oauth-practice-authorization/src/main/kotlin/jp/glory/oauth/practice/authorization/system/spring/SecurityConfig.kt
package jp.glory.oauth.practice.authorization.system.spring

import org.springframework.context.annotation.Configuration
import org.springframework.security.config.annotation.web.builders.HttpSecurity
import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter

@Configuration
class SecurityConfig : WebSecurityConfigurerAdapter(){

    override fun configure(http: HttpSecurity) {
        http.authorizeRequests()
            .anyRequest().permitAll()
            .and()
            .csrf().disable()
    }

}